.github/workflows/pages.yml

```diff
@@ -47,7 +47,7 @@ jobs:
           mkdir ${{github.workspace}}/documentation/imgdoc2net
           cp -r ${{github.workspace}}/libimgdoc2/doxygen/html/* ${{github.workspace}}/documentation/libimgdoc2
           cp -r ${{github.workspace}}/dotnet/imgdoc2Net/doxygen/html/* ${{github.workspace}}/documentation/imgdoc2net
-          echo "<meta http-equiv=\"Refresh\" content=\"0; url='https://ZEISS.github.io/imgdoc2/libimgdoc2/'\" />" >> ${{github.workspace}}/documentation/index.html
+          echo "<meta http-equiv=\"Refresh\" content=\"0; url='libimgdoc2/'\" />" >> ${{github.workspace}}/documentation/index.html
```

Comment:
Did not really manage to test this locally (however the same holds for the original line), so from a semantic point of view, I guess this should work since the relative link should be resolved against the host URL.

Reply:
Yes, I tested this (with my forked repo), and - this let's me test things with the forked repo more easily, since it then is pointing to the "correct" location.